azure-pipelines.yml

```diff
@@ -23,12 +23,18 @@ jobs:
     inputs:
       versionSpec: '$(python.version)'
       architecture: 'x64'
-
+      
+  - script: |
+      /sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16
+      export DISPLAY=:99.0
```

Comment:
The exports do not survive between scripts.

Reply:
Thank you, I will do it in a more robust way